tests/bug-4903/bug-4903-01/test.yaml

```diff
@@ -54,12 +54,8 @@ checks:
 - filter:
     count: 1
     match:
-      dest_ip: 192.168.200.1
```

Comment:
it's unclear why this is removed... did something change in ssh logging?

Reply:
Not yet in https://github.com/OISF/suricata/pull/9797 v13.1 but in v13.4 there is some change

Pushing a new SV PR without this, and rebasing Suricata PR to reference the new SV PR